popochiu/Characters/Goddiu/CharacterGoddiu.gd

```diff
@@ -48,3 +51,9 @@ func play_talk() -> void:
 # Use it to play the grab animation for the character
 func play_grab() -> void:
 	.play_grab()
+
+# Updates character position in the current room if anti-glide animation is on
+# should be called every frame in call method track in animation player
+func update_position():
+	E.current_room.update_characters_position(self)
```

Comment:
Not needed if you move the function to `PopochiuCharacter.gd` as mentioned in the comment for the `CharacterTemplate.gd` file.

This comment also applies for `CharacterPopsy.gd` and `CharacterTrapusinsiu.gd`.